cmd/solana_exporter/utils_test.go

```diff
@@ -63,3 +88,17 @@ func TestGetEpochBounds(t *testing.T) {
 	assert.Equal(t, int64(20), first)
 	assert.Equal(t, int64(29), last)
 }
+
+//go:embed testdata/block-297609329.json
+var blockJson []byte
+
+func TestCountVoteTransactions(t *testing.T) {
+	var block rpc.Block
+	err := json.Unmarshal(blockJson, &block)
+	assert.NoError(t, err)
+
+	voteCount, err := CountVoteTransactions(&block)
+	assert.NoError(t, err)
+	// https://explorer.solana.com/block/297609329
+	assert.Equal(t, 1048, voteCount)
+}
```

Comment:
Can you also test the non-vote transactions via a length check like you did here please?

Reply:
sure can do